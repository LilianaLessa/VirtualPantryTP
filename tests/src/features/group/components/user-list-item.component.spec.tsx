import * as React from "react";
import * as renderer from "react-test-renderer";
import UserListItem from "../../../../../src/features/group/components/user-list-item.component";
import {
  createMockGroup,
  createMockUserInGroup,
} from "../../../../../src/dev-utils";

describe("App", () => {
  it("snapshot test", () => {
    const group = createMockGroup();
    const userInGroup = createMockUserInGroup(group);

    const tree = renderer
      .create(
        <UserListItem
          group={group}
          userInGroup={userInGroup}
          deleteUserInGroupCallback={() => {}}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
