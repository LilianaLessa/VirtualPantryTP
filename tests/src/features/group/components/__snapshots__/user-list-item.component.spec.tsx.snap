// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<View
  style={
    {
      "backgroundColor": "#E6E6E6",
      "borderColor": "#000000",
      "borderWidth": 1,
    }
  }
>
  <View
    style={
      {
        "alignSelf": "stretch",
        "backgroundColor": "#E6E6E6",
        "flexDirection": "row",
      }
    }
  >
    <Text />
    <Text
      style={
        {
          "alignSelf": "stretch",
          "flex": 1,
          "height": 35,
          "margin": 0,
          "marginRight": 5,
        }
      }
    />
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "alignSelf": "stretch",
        "backgroundColor": "#E6E6E6",
        "flexDirection": "row",
        "margin": 0,
        "marginBottom": 5,
        "marginLeft": 5,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "alignSelf": "stretch",
          "backgroundColor": "#E6E6E6",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "flex-start",
        }
      }
    >
      <RCTSwitch
        accessibilityRole="switch"
        disabled={true}
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        onTintColor="rgba(103, 80, 164, 1)"
        style={
          [
            {
              "height": 31,
              "width": 51,
            },
            {
              "height": 23,
              "width": 45,
            },
          ]
        }
        tintColor="rgba(103, 80, 164, 1)"
        value={false}
      />
      <Text
        style={
          {
            "color": "#121212",
          }
        }
      >
        Administrator
      </Text>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "alignSelf": "stretch",
          "backgroundColor": "#E6E6E6",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "flex-start",
        }
      }
    >
      <RCTSwitch
        accessibilityRole="switch"
        disabled={true}
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        onTintColor="rgba(103, 80, 164, 1)"
        style={
          [
            {
              "height": 31,
              "width": 51,
            },
            {
              "height": 23,
              "width": 45,
            },
          ]
        }
        tintColor="rgba(103, 80, 164, 1)"
        value={false}
      />
      <Text
        style={
          {
            "color": "#121212",
          }
        }
      >
        Inviter
      </Text>
    </View>
  </View>
</View>
`;
